app/controllers/sdg_management/local_targets_controller.rb

```diff
@@ -3,12 +3,13 @@ class SDGManagement::LocalTargetsController < SDGManagement::BaseController
 
   LocalTarget = ::SDG::LocalTarget
```

Comment:
This line isn't that useful anymore :thinking:. How about removing it and using `SDG::LocalTarget` in the `create` action?